Background Scripts/listUsersHavingSpecificRole/script.js

```diff
@@ -0,0 +1,11 @@
+function getUsersBasedOnrole(role){
+	var itilUsers = [];
+	var userGR = new GlideRecord('sys_user');
+	userGR.addEncodedQuery('roles.name=itil');
```

Comment:
This should use the function parameter instead of a static value. Additionally, please confirm that dot walking like this works in a list field.